language-support/qute/qute.tmLanguage.json

```diff
@@ -80,17 +86,45 @@
 			"name": "string.unquoted.cdata.qute"
 		},
     "section_start_tag": {
```

Comment:
Could you move this rule after the core rule to respect the same order than patterns include